### DISCUSSION TITLE
Nominating @arjun024 for the role of Maintainer

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#236] [Ruby Team]
I'd like to nominate @arjun024 as a maintainer for the Ruby subteam. 
He has been in the buildpacks space for a while, working on other maintainer groups and I'd love to have his buildpacks expertise on our maintainer team as well.
👍 / 👎 ?

cc/ @paketo-buildpacks/ruby-maintainers
@paketo-buildpacks/steering-committee

**Comment by ForestEckhardt** [comment]
Congrats @arjun024! You have been added to the Ruby Maintainer team.

Please open a PR on the [TEAMS.md](https://github.com/paketo-buildpacks/community/blob/main/TEAMS.md) to add @arjun024 to the list of maintainers.
